### PR TITLE
The login should this time work

### PR DESCRIPTION
**Pull request by maryburrito**

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
-{% endblock %}
-
 {% block content %}
 <h2>Log In</h2>
 <form method="post">
-  {% csrf_token %}
-  {{ form.as_p }}
-  <button type="submit">Log In</button>
+ {% csrf_token %}
+ {{ form.as_p }}
```
Comment (parrotmac) [inline templates/login.html]: Here's the fix for the forms so they use bootstrap (double check this and then commit the result from your computer if this looks right)
```suggestion
 {% bootstrap_form form %}
```

File: backtrack/settings.py [hunk 2]
```diff
@@ -138,3 +138,5 @@
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
 
 LOGIN_URL = '/login'
+LOGIN_REDIRECT_URL = '/students/'
```
Comment (parrotmac) [inline backtrack/settings.py:141]: 🎉

File: tracker/views.py [hunk 3]
```diff
@@ -21,8 +21,12 @@
 from django.contrib.auth.forms import AuthenticationForm
 from django.contrib import messages
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.views import LoginView    
 
+class Login(LoginView):
+    template_name = 'Login.html'
```
Comment (parrotmac) [inline tracker/views.py:27]: What's this about?

**Review by parrotmac — CHANGES_REQUESTED** [review]

File: tracker/views.py (hunk 3 above)
Comment (maryburrito) [reply tracker/views.py:27]: Oh, I need to delete this, I thought that I needed this at one point when I trying to fix the error with the login page. I forgot to take it out

File: templates/login.html [hunk 4]
```diff
@@ -1,14 +1,10 @@
...
-{% endblock %}
-
 {% block content %}
 <h2>Log In</h2>
 <form method="post">
-  {% csrf_token %}
-  {{ form.as_p }}
-  <button type="submit">Log In</button>
+ {% csrf_token %}
+ {% bootstrap_form form %}
```
Comment (parrotmac) [inline templates/login.html:7]: There are some other places where you can make this same change, but no need to do that now if you don't want to.

**Review by parrotmac — APPROVED** [review]
LGTM!

It looks like you might be reusing the same branch, `mp/fixspace` for all your pull requests. I think this is why you were having trouble making a new pull request earlier.

File: templates/login.html (hunk 4 above)
Comment (maryburrito) [reply templates/login.html:7]: OH MY, that makes so much more sense! I was confused why it was all showing up as fixspace!

**Pull request merged**